Makefile

```diff
@@ -59,9 +59,28 @@ prow-jobs:
 	oc process -f cluster/ci/jobs/commenter.yaml | oc apply -f -
 .PHONY: prow-jobs
 
-mungegithub:
+mungegithub: submit-queue-secrets origin-submit-queue installer-submit-queue logging-submit-queue
 .PHONY: mungegithub
 
+submit-queue-secrets:
+	# HMAC_TOKEN is used for encrypting Github webhook payloads.
+	oc create secret generic sq-hmac-token --from-literal=token=${HMAC_TOKEN} -o yaml --dry-run | oc apply -f -
+	# OAUTH_TOKEN is used for manipulating Github PRs/issues (labels, comments, etc.).
+	oc create secret generic sq-oauth-token --from-literal=token=${OAUTH_TOKEN} -o yaml --dry-run | oc apply -f -
```

Comment:
`OAUTH_TOKEN` is already used in prow. `SQ_OAUTH_TOKEN` or `MERGE_TOKEN` or ... ?

Reply:
oh right this _is_ a different OAuth token